python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -505,6 +511,68 @@ def _build_column_lineage_metadata(
             )
 
 
+DbtDagsterEventType = Union[Output, AssetMaterialization, AssetCheckResult, AssetObservation]
+
+
+class DbtPostprocessingAsyncTask(ABC):
```

Comment:
I don't think we would want to generalize to subclass a stateless `AsyncTask`. 

In the future, for efficiency, we would actually want this to be stateful (e.g. to store adapter connections and cached information from API calls).

Let's remove this for now and just have a single function that does post-processing on an event.

Reply:
Yeah, I agree it's probably premature, even if we want to write a few different tasks ourselves I think we don't need the abstraction